dora/integration/fuse/src/main/java/alluxio/fuse/options/FuseOptions.java

```diff
@@ -31,17 +31,17 @@
 public class FuseOptions {
   private static final Logger LOG = LoggerFactory.getLogger(FuseOptions.class);
   public static final PropertyKey FUSE_UPDATE_CHECK_ENABLED =
-      PropertyKey.Builder.booleanBuilder("fuse.update.check.enabled")
+      PropertyKey.Builder.booleanBuilder("alluxio.fuse.update.check.enabled")
```

Comment:
@dbw9580  is this purposely named as `fuse.update.check.enabled` rather than `alluxio.fuse.update.check.enabled` ?

Reply:
The rationale for not defining it in `PropertyKey.java` and not starting with `alluxio.` is that this is purely a command line option, and it should not be made accessible in `alluxio-site.properties`, etc. I named it without the alluxio prefix and put it here so that the usage should be limited to fuse CLI facilities, hopefully.

Reply:
The CLI can be removed and not allow user to disable it from this approach.
do we consider using 
```
dev/scripts/src/alluxio.org/build/profiles.yml:    mvnArgs: "-Dupdate.check.enabled=false"
docs/_data/generated/en/master-configuration.yml:alluxio.master.update.check.enabled:
docs/_data/generated/master-configuration.csv:alluxio.master.update.check.enabled,"true"
dora/core/common/src/main/java-templates/alluxio/ProjectConstants.java:  public static final String UPDATE_CHECK_ENABLED = "${update.check.enabled}";
dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:        "alluxio.master.update.check.enabled";
dora/integration/fuse/src/main/java/alluxio/fuse/options/FuseOptions.java:      PropertyKey.Builder.booleanBuilder("fuse.update.check.enabled")
pom.xml:    <update.check.enabled>true</update.check.enabled>
```
the update check maven arguments way for this fuse update check disable during internal development and testing?

Reply:
i am also worried about hardcoding this. there needs to be a way to differentiate whether or not the update checks should run by default. the current mechanism is to set `-Dupdate.check.enabled=false` when building the code for specific purposes; see https://github.com/Alluxio/alluxio/blob/672562bc96b3bb575ad66e3d8e765ca2794b089e/dev/scripts/src/alluxio.org/build/profiles.yml#L127C1-L127C1

i recommend reusing the same mechanism and set `.setDefaultValue(Boolean.parseBoolean(ProjectConstants.UPDATE_CHECK_ENABLED))` for this property

Reply:
followed what @Xenorith suggested